### PR TITLE
Scoreboard tweak

### PR DESCRIPTION
**Pull request by adacovsk**
[hotfix][tweak]

Removed some obsolete commented code and "panacea" which is almost impossible to get, and added one line for disease_good (which was already used, but not added to scoreboard).

### CODE REVIEW

**Comment by Eneocho** [comment]
Panacea is when someone makes a complete vaccine for all antibodies?

**Comment by adacovsk** [comment]
> Panacea is when someone makes a complete vaccine for all antibodies?

no, it's when you get a single certain arbitrary score for a disease, based on "the angel number"

File: code/datums/gamemode/dynamic/dynamic.dm [hunk 1]
```diff
@@ -161,6 +160,7 @@ var/stacking_limit = 90
 				ruletype = "midround"
 			dat += "([ruletype]) - <b>[DR.name]</b>[DR.calledBy ? " (called by [DR.calledBy])" : ""]<br>"
 			rules_text += "[ruletype] - **[DR.name]** [DR.calledBy ? " (called by [DR.calledBy])" : ""]"
+		//dat += "<a href='?src=\ref[src];threatlog=1'>\[View Log\]</a>"
```
Comment (DamianX) [inline code/datums/gamemode/dynamic/dynamic.dm]: See https://github.com/vgstation-coders/vgstation13/blob/b30153715d75e28f0b7278c569108eb3c63ec5e6/code/datums/gamemode/dynamic/dynamic.dm#L110

Why would you remove this?

Comment (adacovsk) [reply code/datums/gamemode/dynamic/dynamic.dm]: I thought it was just for admins, and also that it didn't work. Then I tested it and it did work (for admins) and I didn't update it

**Pull request merged**